### ISSUE TITLE
ReadTheDocs documentation

### ISSUE THREAD

**Issue by LecrisUT** [beakerlib/libraries#13]
ReadTheDocs supports [subprojects](https://docs.readthedocs.io/en/stable/subprojects.html), meaning that the rtd pages built on this repo can be custom pages on either beakerlib.readthedocs.io or tmt.readthedocs.io.

This is helpful because there will be a central place to find all of the libraries.

Pinging: @lukaszachy @psss @sopos

**Comment by psss** [comment]
Nice idea! Sounds good to me to provide the list of available `beakerlib` libraries directly in the documentation. I'd say beakerlib.readthedocs.io would be a better choice for the lib details and add a link to the page from the `tmt` documentation.

**Comment by LecrisUT** [comment]
Great to hear :). Can you enable RTD on tmt PRs so I can test the integration later on?

**Comment by psss** [comment]
This is done using the github action?
https://github.com/marketplace/actions/sphinx-docs-build-with-readthedocs-docker

**Comment by LecrisUT** [comment]
No, it's in the [RTD configuration](https://readthedocs.org/dashboard/tmt) of the tmt project. I am still trying to get RTD to work with GH actions: https://github.com/readthedocs/actions/issues/27.


<details><summary> Read the docs configuration page</summary>

![Screenshot_20230621_150256](https://github.com/beakerlib/libraries/assets/25577083/1899d94e-0cc6-4030-b2b8-791952dd9091)

</details>

**Comment by psss** [comment]
Thanks for the pointer. Should be enabled now.

**Comment by LecrisUT** [comment]
Oh I forgot to mention (had this happen to me several times), don't forget to hit save :D.

**Comment by psss** [comment]
> Oh I forgot to mention (had this happen to me several times), don't forget to hit save :D.

Hm, I've updated, and saved, but no new status appears in the fresh pull requests. Not even after adding new commits. Is there anything missing in the `tmt` repo?

**Comment by LecrisUT** [comment]
I think it doesn't add for current PRs unless they are closed and reopened. But I tried that with https://github.com/teemtee/tmt/pull/2098 and I don't see any changes [here](https://readthedocs.org/projects/tmt/builds/). The only other thing I can think of is to check the webhook settings and recent deliveries there. Make sure `pull_request` events are sent.

**Comment by psss** [comment]
> The only other thing I can think of is to check the webhook settings and recent deliveries there. Make sure `pull_request` events are sent.

Yes, that was it. Thanks for the hint!

**Comment by LecrisUT** [comment]
Great, it doesn't even require closing and reopening PRs.
